mlrun/runtimes/pod.py

```diff
@@ -175,6 +178,28 @@ def _set_volume_mount(self, volume_mount):
         self._volume_mounts[volume_mount_key] = volume_mount
 
 
+class AutoMountType(str, Enum):
+    none = "none"
+    auto = "auto"
+    v3io_cred = "v3io_cred"
+    v3io_fuse = "v3io_fuse"
+    pvc = "pvc"
+
+    @classmethod
+    def _missing_(cls, value):
+        return AutoMountType.auto
```

Comment:
Just to make it more explicit and understand-able what's going on let's add
```
def default():
        return AutoMountType.auto
```
and use it in this method